tests/prueba1.js

```diff
@@ -0,0 +1,12 @@
+describe('Login', () => {
+    it('Deberia iniciar sesion', async ()=> {
+    await browser.url('bank');
+    let username = await $('#username');
+    await username.setValue('federicodelucchi');
+    let password = await $('#password');
+    await password.setValue('12345678');
+    let btn = await $('#submit');
+    await btn.keys('Enter');
+    await expect(await $('span').getText('error'));
```

Comment:
Prefer using  native **wdio matchers** for improved readability. In this case, `toHaveTextContaining()` looks like the most relevant for asserting what you want. It'll look something like:

```javascript
await expect(await $('span')).toHaveTextContaining('error');
```

Also, notice that the syntax for `expect()` statements is something like: 
```javascript
await expect(value).assertionMethod(expectedValue)
```
In your current implementation, you only provided a `value` to the `expect()` function (as `getText()` is just an element's method for retrieving its text, not an assertion method on the expect() function, and it takes no parameter, so the "error" string is being ignored in this context, it only retrieves whatever text the element has). You were not really asserting anything there.

Referece: https://webdriver.io/docs/api/expect-webdriverio/#tohavetext